.travis.yml

```diff
@@ -12,7 +12,5 @@ cache:
 install: true
 
 script:
-  - find ./ -name "pom.xml" -type f -exec mvn xml:check-format -f {} \;
-  - find ./ -name "pom.xml" -type f -exec mvn formatter:validate -f {} \;
-  - ./mvn-build-all.sh clean install jacoco:report
+  - ./mvn-build-all.sh xml:check-format formatter:validate clean install jacoco:report
```

Comment:
- [ ] I think it's better to divide `xml:check-format formatter:validate` and `install jacoco:report`.
Because, all commands are executed each projects, so even if there is a format problem in one project, other projects may be installed.